src/tests/routes/test_devices.py

```diff
@@ -9,12 +9,13 @@
 
 from app import db
 from app.api import crud, security
-from tests.db_utils import get_entry, fill_table
+from tests.db_utils import get_entry, fill_table, TestSessionLocal
 from tests.utils import update_only_datetime, parse_time
 
 USER_TABLE = [
     {"id": 1, "login": "first_login", "access_id": 1, "created_at": "2020-10-13T08:18:45.447773"},
     {"id": 2, "login": "second_login", "access_id": 2, "created_at": "2020-11-13T08:18:45.447773"},
+    {"id": 3, "login": "sixth_login", "access_id": 5, "created_at": "2020-11-13T08:18:45.447773"},
```

Comment:
This entry and the one you added in the device table can't share the same access_id